lib/resources/majors.tsx

```diff
@@ -2653,6 +2860,350 @@ const bsMolCell: Major = {
   ],
 };
 
+// https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/mechanical-engineering/mechanical-engineering-bachelor-science/#requirementstext
+/**
+ * Problem 1: Same issue with relative exclusivity
+ * Solution: Exclusivity should be relative to distributions and individual fine requirements rather than just be a boolean. Also, when a distribution is satisfied, it should no longer be considered in the exclusivity check.
+ */
+const bsMechE: Major = {
+  degree_name: 'B.S. Mechanical Engineering',
+  abbrev: 'B.S. MechE',
+  department: 'EN Mechanical Engineering',
+  total_degree_credit: 126,
+  wi_credit: 6,
+  url: 'https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/mechanical-engineering/mechanical-engineering-bachelor-science/#requirementstext',
+  distributions: [
+    {
+      name: 'Mathematics',
+      required_credits: 16,
+      min_credits_per_course: 4,
+      description:
+        'The student must complete one of the tracks of mathematics courses, offered ' +
+        'either by the Mathematics department in the Kreiger School of Arts and Sciences ' +
+        'or the Applied Mathematics and Statistics department in the Whiting School of Engineering.',
+      criteria:
+        'AS.110.108[C]^OR^AS.110.109[C]^OR^AS.110.113[C]^OR^AS.110.202[C]^OR^EN.553.291[C]^OR^AS.110.201[C]^OR^AS.110.212[C]^OR^AS.110.302[C]',
+      pathing: 1,
+      fine_requirements: [
+        {
+          description:
+            '<b>4 Math Courses</b> <br />' +
+            'Complete all of the following: <br />' +
+            'AS.110.108 Calculus I (Physical Sciences & Engineering) <br />' +
+            'AS.110.109 Calculus II (Physical Sciences & Engineering) <i>OR</i> AS.110.113 Honors Single Variable Calculus <br />' +
+            'AS.110.202 Calculus III <br />' +
```

Comment:
I thought of doing that, but then that would mean we would have Calc I, II and III as one distribution, and then a whole other distribution just for the linear algebra and diffEQ pathing, because pathing exists only at the distribution level. I thought that would look a little awkwardly placed.

Reply:
I wouldn't be pathing, there would just be a `Linear Algebra and Differential Equations` fine requirement with the description `Select one of the following: <br /> LADE OR (Linear Algebra AND DiffEq)`. Basically trust the student to do take a valid combination, but yeah this fine requirements would fulfilled with Linear Algebra, when it shouldn't. 

tough call

Reply:
The way it is enforced right now, it will make sure that the student is not required to make the call. This sort of guarantees that the student chooses one path or the other.